x/ibc/03-connection/types/codec.go

```diff
@@ -11,11 +11,11 @@ import (
 // Any.
 func RegisterInterfaces(registry codectypes.InterfaceRegistry) {
 	registry.RegisterInterface(
-		"cosmos_sdk.ibc.v1.connection.ConnectionI",
+		"ibc.core.connection.v1.ConnectionI",
```

Comment:
not sure if we should rename this to just `Connection`

Reply:
I'd be fine renaming. `exported.Connection` should imply it is an interface

Reply:
though ideally we would do this with all interfaces since it seems the channel has the same format